### ISSUE TITLE
Docker Kafka container restarting on Mac intel processor

### ISSUE THREAD

**Issue by avgupta456** [getsentry/sentry#29022]
### Environment

self-hosted (`onpremise` deployment)

### Version

21.10.0.dev0

### Steps to Reproduce

1. Clone from GitHub `master` branch
2. Follow instructions at https://develop.sentry.dev/environment/ to setup environment
3. `sentry upgrade`, `sentry devservices up` `sentry devserver --workers`
4. Observe Docker containers, specifically sentry_kafka

Using MacBook Pro 2020 w/ Intel Core, Docker v20.10.8

### Expected Result

Containers are all running and user is able to navigate to localhost:8000 and view new errors for current projects.

### Actual Result

The Kafka container is repeatedly restarting. When I trigger an event for a project, I do not see any events on the issues dashboard.

<img width="1054" alt="Screen Shot 2021-10-01 at 5 21 05 PM" src="https://user-images.githubusercontent.com/16708871/135687659-d6fe8ca5-b255-4546-aeff-68cf9cba416f.png">

**Comment by armenzg** [comment]
Hi @avgupta456 I've noticed that your version says `5.1.2`, however, we're now using 6.2.0; would you mind stopping the containers, deleting the old container and try again?

**Comment by getsentry-release** [comment]
Routing to @getsentry/open-source for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

**Comment by avgupta456** [comment]
@armenzg sorry I think I was testing reverting the version update in case that fixed anything. Attached is a screenshot showing the same problem with version `6.2.0`

<img width="1049" alt="Screen Shot 2021-10-04 at 12 58 37 PM" src="https://user-images.githubusercontent.com/16708871/135893351-535ca914-44b4-48d4-8526-0c54634bc741.png">

**Comment by BYK** [comment]
I mean the error seems pretty clear: the data volume is not writable?

Also, if you can share text logs instead of screenshots, it would be a lot easier for us to dig through them.

**Comment by avgupta456** [comment]
Here are the text logs

```
===> User

uid=1000(appuser) gid=1000(appuser) groups=1000(appuser)

===> Configuring ...

===> Running preflight checks ... 

===> Check if /var/lib/kafka/data is writable ...

===> User

uid=1000(appuser) gid=1000(appuser) groups=1000(appuser)

===> Configuring ...

===> Running preflight checks ... 

===> Check if /var/lib/kafka/data is writable ...

===> User

uid=1000(appuser) gid=1000(appuser) groups=1000(appuser)

===> Configuring ...

===> Running preflight checks ... 

===> Check if /var/lib/kafka/data is writable ...

===> User

uid=1000(appuser) gid=1000(appuser) groups=1000(appuser)

===> Configuring ...

Command [/usr/local/bin/dub path /var/lib/kafka/data writable] FAILED !

Command [/usr/local/bin/dub path /var/lib/kafka/data writable] FAILED !

Command [/usr/local/bin/dub path /var/lib/kafka/data writable] FAILED !
```

**Comment by armenzg** [comment]
@BYK We already landed a fix that worked on Apple M1 yet it does not seem to work on Intel. On an initial investigation, I noticed that I can start the image with the commands listed [here](https://github.com/getsentry/sentry/pull/28724#issuecomment-924021823) yet `sentry devservices up kafka` fails to do it properly.

I'm leaning towards backing my code out and investigate later. Dealing with the ClickHouse image not working is currently the big fish to fry.
